README.md

```diff
@@ -1,48 +1,55 @@
-You will need to supply your own Discord Bot Token (found at https://discord.com/developers/), Client ID (found once you create a Discord Application), and Guild ID (found when you enable Developer Mode in Discord Settings in the Main Client, then copy your server, which is also known as a guild, ID). Other than that, this Dockerfile is usable OOTB (out of the box).
+You need to supply your own
+1. Discord Bot Token (found at https://discord.com/developers/)
+2. Client ID (found once you create a Discord Application)
+3. Guild ID (found when you enable Developer Mode in Discord Settings in the Main Client
```

Comment:
Merge lines 4 and 5 up to the, "...guild, ID)." and remove the comma after guild and the period after the closed parenthetical.